wallets/provider-all/src/index.ts

```diff
@@ -52,6 +62,21 @@ export const allProviders = (options?: Options) => {
     }
   }
 
+  if (
+    !isWalletExcluded(providers, {
+      type: WalletTypes.TREZOR,
+      name: 'Trezor',
+    })
+  ) {
+    if (!!options?.trezor?.manifest) {
+      trezor.init(options.trezor);
+    } else {
+      throw new Error(
+        'Trezor has been included in your providers. Passing a manifest is required. Make sure you are passing "trezorManifest".'
+      );
+    }
+  }
```

Comment:
Please make sure trezor config is not required. It is not backward compatible.

Reply:
fixed